js-packages/@prestojs/rest/src/Paginator.ts

```diff
@@ -17,14 +17,29 @@ export default class Paginator<State, InternalState> implements PaginatorInterfa
     setCurrentState: (set: State) => void;
     setInternalState: (set: InternalState) => void;
 
+    constructor(currentStatePair = null, internalStatePair = null) {
```

Comment:
There should be some documentation explaining what these are (even if it's just a `@see ...` ). In particular, is it documented somewhere that the currentStatePair can be used to specify whether to store state in a hidden way or  to store it in the URL? If so, refer back to that